src/js/components/Video.js

```diff
@@ -152,19 +205,28 @@ export default class Video extends Component {
       );
     }
 
+    var onClickControl = this._onClickControl;
+    if (this.props.onClick) {
+      onClickControl = this.props.onClick;
+    }
+
```

Comment:
can we simplify this:

`const onClickControl = this.props.onClick || this._onClickControl;`